### DISCUSSION TITLE
Nhost Angular SDK

### DISCUSSION THREAD

**Discussion by zuohuadong** [nhost/nhost#998] [Ideas]
Can nhost support angular?

**Comment by kornect** [comment]
Just started checking it out and it would be really awesome if angular support is added

EDIT: Got it working with angular by using the base library (instructions on the repo readme). it will still be useful to have a dedicated one but this should get you going

**Reply by kornect** [reply]
Started getting an error with the angular setup, something to do with the FormData import. Logged an issue here #925

**Comment by szilarddoro** [comment]
We don't plan to create an official `@nhost/angular` package in the near future.

Although as @kornect mentioned, you can add `@nhost/nhost-js` to your Angular project and use all of Nhost's features without limitations.

**Comment by Parables** [comment]
@szilarddoro  @elitan  Would you accept and merge PRS for other frameworks?

**Comment by elitan** [comment]
If you want to work on an Angular SDK I'd recommend doing it in a private repo initially and we can move it to our official repo once at a later stage. 

I'm moving this issue to a Discussion.

**Comment by oriollpz** [comment]
It would be awesome to make it work with Angular...

**Comment by kuncevic** [comment]
what would be the benefit having dedicated package vs using @nhost/nhost-js ?

**Reply by KerickHowlett** [reply]
The biggest reason I can think of is the ability to make full use of RxJS for where it's most appropriate, which Angular uses quite extensively throughout its ecosystem and underlining infrastructure. 

Admittedly, I haven't used the SDK yet and haven't looked too deeply into it, so I could be missing something. Or maybe it just doesn't need it on account of using GraphQL. _(I'm still BaaS shopping. 😅)_
